src/Products/ZopeVersionControl/Version.py

```diff
@@ -12,14 +12,14 @@
 ##############################################################################
 
 import time
+from io import BytesIO
 
 from AccessControl import ClassSecurityInfo
 from AccessControl.class_init import InitializeClass
 from Acquisition import Implicit
 from Acquisition import aq_base
 from OFS.SimpleItem import SimpleItem
 from Persistence import Persistent
-from ZODB._compat import BytesIO
 from ZODB._compat import Pickler
```

Comment:
This will probably break with the next ZODB release. Importing from modules starting with an underscore can be a source of trouble. But we'll see and fix it later.

Reply:
This underscores the need for using tools like `zope.deprecation` more often.